### PR TITLE
Django 1.8 merge

### PR DESCRIPTION
**Pull request by kluo**
@dcadams

### CODE REVIEW

File: openassessment/templates/openassessmentblock/peer/oa_peer_turbo_mode.html [hunk 1]
```diff
@@ -52,16 +52,12 @@ <h3 class="message__title">{% trans "Peer Assessments Complete" %}</h3>
...
-                                {% if track_changes %}<input id="track_changes_clear_button" type="button" value="Clear Changes" />{% endif %}
                                 <h3 class="peer-assessment__display__title">{% trans "Assessment #" %}
                                     <span class="peer-assessment__number--current">{{ review_num }}</span> of
                                     <span class="peer-assessment__number--required">{{ must_grade }}</span>
                                 </h3>
                             </header>
 
-                            <div class="peer-assessment__display__response" {% if track_changes %} id="track-changes-content" contenteditable="true" {% endif %}>
-                                {{ peer_submission.answer.text|linebreaks }}
-                            </div>
```
Comment (dcadams) [inline openassessment/templates/openassessmentblock/peer/oa_peer_turbo_mode.html]: Is this functionality (clear button and track changes text box) being removed?

**Comment by dcadams** [comment]
@kluo I'm seeing errors with ORA2 problems. Can you let me know when it's ready to be eyeballed.

File: openassessment/templates/openassessmentblock/peer/oa_peer_turbo_mode.html (hunk 1 above)
Comment (kluo) [inline openassessment/templates/openassessmentblock/peer/oa_peer_turbo_mode.html]: No, it's been factored out into `openassessmentblock/oa_submission_answer.html`

**Comment by kluo** [comment]
retest this please

**Comment by kluo** [comment]
Superceded by #21

**Pull request closed without merging**